SabreTools.DatTools/Rebuilder.cs

```diff
@@ -226,7 +227,17 @@ public class Rebuilder
 
                     // If we are supposed to delete the file, do so
                     if (delete && rebuilt)
-                        File.Delete(input);
+                    {
+                            try 
```

Comment:
These `try/catch` blocks look near identical. It may be worthwhile to abstract it out into a helper method within this file so you can do a single-line invoke of that code instead.